### PR TITLE
cli interaction working, but not yet saving to DB

### PR DESCRIPTION
**Pull request by danirenae**
# Description
This pull request has updated tests for get_all_payments and save_payment. This pull request also has all cli functionality for payment.

# Related Ticket(s)
This is related to issue number #16 

## Testing

[x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x] I certify that all existing tests pass

The changes were made in test_payment.py also payment_cli.py was created.

##How to test
1. get fetch
2. get checkout da-cli
3. go into tests file and run:
``` python test_payment.py ```

4. go into cli file and run:
``` python payment_cli.py ```

### CODE REVIEW

**Comment by juliakimchung** [comment]
You can delete""""if __name__ == '__main__':
    enter_payment = PaymentSelection()
    enter_payment.run() """ from the payment_cli.py since it will be run from main.py 
Instead, add """enter_payment = PaymentSelection()
            enter_payment.run()""" under the' if selected statement' for the payment in the main.py

File: cli/payment_cli.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import sys
+sys.path.append("../")
+from db.payment_db_interactor import *
+from models.payment import *
+
+class PaymentSelection():
+    """Class to contain the Bangazon Payment Selection view. After running this file the PaymentSelection run method is called.
```
Comment (samphillips1879) [inline cli/payment_cli.py:7]: Good doc strings

File: tests/test_payment.py [hunk 2]
```diff
@@ -46,9 +46,41 @@ def test_payment_can_be_saved(self):
...
         saved_payment = PaymentDatabaseInteractor()
         saved_payment.save_payment(self.juan_payment)
-        self.assertIn((4, "12345678", "Visa", 1), saved_payment.get_all_payments())
+        data = saved_payment.get_all_payments()
 
+        self.assertIsInstance(data, list)
+        test_index = len(data)
+        payment_present = False
 
+        for payment in saved_payment.get_all_payments():
```
Comment (samphillips1879) [inline tests/test_payment.py:55]: Nice test! 😉

**Comment by danirenae** [comment]
@juliakimchung: @samphillips1879 will be adding all cli components to main.py

**Comment by pinkston4** [comment]
👍 all 5 tests passed, cli worked

**Review by samphillips1879 — COMMENTED** [review]
Tests pass on my machine, new payment is present in database! looks good to go 👍

**Pull request merged**